akka-cluster-tools/src/main/scala/akka/cluster/pubsub/DistributedPubSubMediator.scala

```diff
@@ -84,13 +85,15 @@ object DistributedPubSubSettings {
  * @param removedTimeToLive Removed entries are pruned after this duration
  * @param maxDeltaElements Maximum number of elements to transfer in one message when synchronizing
  *   the registries. Next chunk will be transferred in next round of gossip.
+ * @param sendToDeadLettersWhenNoSubscribers When a message is published to a topic with no subscribers send it to the dead letters.
  */
 final class DistributedPubSubSettings(
-  val role:              Option[String],
-  val routingLogic:      RoutingLogic,
-  val gossipInterval:    FiniteDuration,
-  val removedTimeToLive: FiniteDuration,
-  val maxDeltaElements:  Int) extends NoSerializationVerificationNeeded {
+  val role:                               Option[String],
+  val routingLogic:                       RoutingLogic,
+  val gossipInterval:                     FiniteDuration,
+  val removedTimeToLive:                  FiniteDuration,
+  val maxDeltaElements:                   Int,
+  val sendToDeadLettersWhenNoSubscribers: Boolean) extends NoSerializationVerificationNeeded {
```

Comment:
The new parameter must be added in a binary backwards compatible way, i.e. add another (deprecated) constructor corresponding to the old constructor